test/parse.js

```diff
@@ -70,3 +70,24 @@ test('assign only once', function() {
   assert.deepEqual({ foo: '', bar: 'bar' },
     cookie.parse('foo=;bar=bar;foo=boo'));
 });
+
+test('multiValuedCookies flag', function () {
+  assert.deepEqual(
+    { foo: ["%1", "boo"], bar: "bar" },
+    cookie.parse("foo=%1;bar=bar;foo=boo", {
+      multiValuedCookies: true
+    })
+  );
+  assert.deepEqual(
+    { foo: ["", "boo"], bar: "bar" },
+    cookie.parse("foo=;bar=bar;foo=boo", {
+      multiValuedCookies: true,
+    })
+  );
+  assert.deepEqual(
+    { foo: ["%1", "boo", "bar"], bar: "bar" },
+    cookie.parse("foo=%1;bar=bar;foo=boo;foo=bar", {
+      multiValuedCookies: true,
+    })
+  );
+});
```

Comment:
Can you add a test for the expected output when there is just a single value for a cookie with this option on?